api/lib/domain/services/user-reconciliation-service.js

```diff
@@ -167,10 +167,30 @@ async function createUsernameByUser({ user: { firstName, lastName, birthdate },
   return username;
 }
 
+async function isSupernumerarySchoolingRegistrationExisting({
```

Comment:
ça serait pas plutôt `doesSupernumerarySchoolingRegistrationExist` ?

Reply:
c'est pareil non ?

Reply:
J'ai regardé sur linguee j'ai pas trop de résultats avec is exist donc je pars sur le does

Reply:
done

Reply:
c'était "is existing" ;)